Gaijinizer patch/Enemies.txt

```diff
@@ -1717,11 +1715,11 @@ After making the other party fall, I slowly take it.#MTLed
 この形態で頭を直接覆い、一気に『洗脳』してしまえば
 より深く、より思い通りの眷族が短時間で誕生する
 > CONTEXT: Enemies/565/BookDescription/3
-Her lower body tentacles are just touching
-Can affect the other's thinking,
-If you cover your head directly in this form and do
-"brainwashing" at once
-Deeper, more fancy tribes are born in a short time#MTLed
+Just being touched by one of the tentacles on her
+lower half is enough for her to influence her
+opponent. But when their head is covered in them
+this allows her to thoroughly brainwash him and
+gain a follower in a very short space of time.#Ron
 > END STRING
```

Comment:
Minor flow:
>_**opponent. But when their head is covered in them,
this allows her to intensely brainwash them and
create a follower in a very short space of time.**_